Taggers/python/diphotonDumpConfig_cff.py

```diff
@@ -11,7 +11,7 @@
     categories = cms.VPSet(),
 
     workspaceName = cms.untracked.string("cms_hgg_$SQRTS"),
-    nameTemplate = cms.untracked.string("$PROCESS_$SQRTS_$LABEL_$SUBCAT_$SYST"),
+    nameTemplate = cms.untracked.string("$PROCESS_$SQRTS_$CLASSNAME_$SUBCAT_$SYST"),
```

Comment:
I think that $CLASSNAME should be taken out here, as the dumper is not class-based.